### PR TITLE
 CQ-4345916 Capture DOR specific properties for the core components

### PR DESCRIPTION
**Pull request by devgurjar**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/core/src/test/resources/form/checkbox/exporter-checkbox.json [hunk 1]
```diff
@@ -12,7 +12,10 @@
     "af:layout" : {
       "tooltipVisible": false,
       "orientation" : "horizontal"
-    }
+    },
+    "fd:dor":{
+      "dorExclusion":false,
+      "dorColspan":null}
```
Comment (nit23uec) [inline bundles/core/src/test/resources/form/checkbox/exporter-checkbox.json:18]: minor: indentation of` }`

File: bundles/core/src/test/resources/form/dropdown/exporter-multiselect-dropdown.json [hunk 2]
```diff
@@ -11,7 +11,10 @@
   "properties": {
     "af:layout" : {
       "tooltipVisible": false
-    }
+    },
+    "fd:dor":{
+      "dorExclusion":false,
+      "dorColspan":null}
```
Comment (nit23uec) [inline bundles/core/src/test/resources/form/dropdown/exporter-multiselect-dropdown.json:17]: pls correct indentation of } everywhere

File: bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Base.java [hunk 3]
```diff
@@ -360,6 +362,19 @@ default Map<String, Object> getProperties() {
...
 
+    /**
+     * Custom DOR Properties
+     *
+     * @since com.adobe.cq.forms.core.components.models.form 1.1.0
+     */
+
+    @JsonIgnore
+    @JsonView(Views.Author.class)
+    @JsonInclude(Include.NON_EMPTY)
```
[the remark this answers is not in the document]
Comment (devgurjar) [reply bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Base.java:372]: Incorporated

File: bundles/core/src/test/resources/form/checkbox/exporter-checkbox.json (hunk 1 above)
Comment (devgurjar) [reply bundles/core/src/test/resources/form/checkbox/exporter-checkbox.json]: Incorporated

File: bundles/core/src/test/resources/form/dropdown/exporter-multiselect-dropdown.json (hunk 2 above)
Comment (devgurjar) [reply bundles/core/src/test/resources/form/dropdown/exporter-multiselect-dropdown.json]: Incorporated

File: bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Base.java [hunk 4]
```diff
@@ -360,6 +362,18 @@ default Map<String, Object> getProperties() {
         return Collections.emptyMap();
     }
 
+    /**
+     * Custom DOR Properties
+     *
+     * @since com.adobe.cq.forms.core.components.models.form 1.1.0
```
Comment (rismehta) [inline bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Base.java:368]: This should be 2.1.0, please re-sync with latest

File: bundles/core/src/test/resources/form/datepicker/exporter-datepicker.json [hunk 5]
```diff
@@ -11,6 +11,10 @@
   "properties": {
     "af:layout" : {
       "tooltipVisible": false
+    },
+    "fd:dor": {
+      "dorExclusion":false,
+      "dorColspan":null
```
Comment (rismehta) [inline bundles/core/src/test/resources/form/datepicker/exporter-datepicker.json:17]: Why are we including null. We should only include if non empty

**Review by rismehta — DISMISSED** [review]
minor comments

**Review by nit23uec — APPROVED** [review]

File: bundles/core/src/test/resources/form/datepicker/exporter-datepicker.json (hunk 5 above)
Comment (devgurjar) [reply bundles/core/src/test/resources/form/datepicker/exporter-datepicker.json]: Incorporated

**Comment by nit23uec** [comment]
@rismehta , can you pls approve this? looking to merge this today.

**Review by rismehta — COMMENTED** [review]
Approved

**Pull request merged**